src/_scss/pages/dashboard/_chooseFilters.scss

```diff
@@ -0,0 +1,25 @@
+.dashboard-choose-filters-flex {
+    height: 100%;
+    display: flex;
+    //display: -ms-flex;
+    //display: -webkit-flex;
+
+    .dashboard-choose-filters {
+        width: 100%;
+        margin: rem(50) rem(100);
+        padding-top: rem(225);
```

Comment:
This would need to be adjusted to keep the message vertically centered as more elements are added to the sidebar. To automatically center it, we could look into using the flex property `align-items: center;` (or `align-self: center;`)

Reply:
Part of the design review was actually that the text didn't move when the size of the sidebar changed so I intentionally kept it from aligning center.